package.json

```diff
@@ -1,24 +0,0 @@
-{
-  "name": "wp-contributions",
-  "version": "1.1.0",
-  "description": "Add your WordPress contributions to your website in a widget or using a template tag.  You can feature your plugin, theme, core contributions, or codex contributions.",
-  "author": {
-    "name": "WebDevStudios",
-    "url": "http://webdevstudios.com"
-  },
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/WebDevStudios/WP-Contributions/"
-	},
-  "license": "GPLv2",
-  "devDependencies": {
-    "grunt": "^1.0.1",
-    "grunt-contrib-cssmin": "^1.0.1",
-    "grunt-contrib-jshint": "^0.11.1",
-    "grunt-contrib-uglify": "^0.8.0",
-    "grunt-contrib-watch": "^1.0.0",
-    "grunt-sass": "^1.2.0",
-    "grunt-wp-i18n": "^0.5.4",
-    "load-grunt-tasks": "^3.5.0"
-  }
-}
```

Comment:
Looks like we had a root level package.json file available originally, but it's getting removed completely in favor of perhaps the one for the block specifically?

Reply:
it was an intent to move it to the root folder, I will order all